mne/decoding/transformer.py

```diff
@@ -137,13 +137,13 @@ def inverse_transform(self, epochs_data, y=None):
             raise ValueError("epochs_data should be of type ndarray (got %s)."
                              % type(epochs_data))
 
-        X = np.atleast_3d(epochs_data)
+        X = np.atleast_3d(epochs_data).copy()
 
         for key, this_pick in six.iteritems(self.picks_list_):
             if self.with_mean:
-                X[:, this_pick, :] += self.ch_mean_[key]
-            if self.with_std:
                 X[:, this_pick, :] *= self.std_[key]
+            if self.with_std:
+                X[:, this_pick, :] += self.ch_mean_[key]
```

Comment:
The intent was to scale first and then center (if both scaling and centering are required), but now the conditions and operations do not match (ie. conditions should have been swapped too).